cppwinrt/code_writers.h

```diff
@@ -2986,13 +2986,15 @@ struct __declspec(empty_bases) produce_dispatch_to_overridable<T, D, %>
             return;
         }
 
+        auto is_opt_type = settings.component_opt && settings.component_filter.includes(type);
```

Comment:
is_component_type?

Reply:
It's "is optimized component type". non-optimized components still can use auto.